src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/JpaInductionPersistenceAdapter.kt

```diff
@@ -23,12 +28,30 @@ class JpaInductionPersistenceAdapter(
   private val inductionMapper: InductionEntityMapper,
   private val previousQualificationsRepository: PreviousQualificationsRepository,
   private val previousQualificationsMapper: PreviousQualificationsEntityMapper,
+  private val noteRepository: NoteRepository,
 ) : InductionPersistenceAdapter {
 
   @Transactional
   override fun createInduction(createInductionDto: CreateInductionDto): Induction {
     val inductionEntity = inductionRepository.saveAndFlush(inductionMapper.fromCreateDtoToEntity(createInductionDto))
     val previousQualificationsEntity = createOrUpdatePreviousQualifications(createInductionDto)
+
+    val noteEntity = createInductionDto.note?.let {
+      noteRepository.saveAndFlush(
+        NoteEntity(
+          reference = UUID.randomUUID(),
+          prisonNumber = createInductionDto.prisonNumber,
+          content = it,
+          noteType = NoteType.INDUCTION,
+          entityType = EntityType.INDUCTION,
+          entityReference = inductionEntity.reference!!,
+          createdAtPrison = inductionEntity.createdAtPrison ?: "N/A",
```

Comment:
Although these should be populated the data model as it stands has these as nullable.